www/docs/intro.md

```diff
@@ -376,6 +376,14 @@ Devnet offers [empty block creation](https://shard-labs.github.io/starknet-devne
 const emptyBlock = await starknet.devnet.createBlock();
 ```
 
+#### Mint tokens to an account
+
+DevNet allows [minting token](https://shard-labs.github.io/starknet-devnet/docs/guide/mint-token#mint-with-a-transaction). You can call `starknet.devnet.mint` like this,
```

Comment:
As much as writing it as DevNet makes sense, that's not how we write it anywhere else, so let's stick to Devnet (lowercase n), as Nathan has [already pointed out](https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/285#discussion_r1056127679)

Reply:
Haha, oops... 🤣 😳